### PR TITLE
Eliminate noisy test output.

### PR DESCRIPTION
**Pull request by rictic**
When there are no unexpected errors, test output should only contain the names of tests.

Also, ignore `not-loadable` warnings, as polymer-build should not load remote resources.

 - [x] CHANGELOG.md has been updated

### CODE REVIEW

File: src/test/analyzer_test.ts [hunk 1]
```diff
@@ -338,7 +338,7 @@ suite('Analyzer', () => {
       ],
       sources: ['a.html', 'b.html', 'entrypoint.html'],
     });
-    const analyzer = new BuildAnalyzer(config);
+    const analyzer = new BuildAnalyzer(config, null);
```
Comment (usergenic) [inline src/test/analyzer_test.ts]: Why can't we rely on default instead of these explicit nulls?

Comment (rictic) [reply src/test/analyzer_test.ts]: Explicit null is different. It's explicitly saying "no output please".

`undefined` is what you pass in if you want the default.

Comment (usergenic) [reply src/test/analyzer_test.ts]: Or to put it another way, why can't it default to null?

Comment (usergenic) [reply src/test/analyzer_test.ts]: OH ... sorry I see why.

**Comment by usergenic** [comment]
Lets merge this before the 3.0.0-pre.1 release.

**Pull request merged**